queue_services/entity-bn/flags.json

```diff
@@ -0,0 +1,17 @@
+{
+    "flagValues": {
+        "db-versioning": {
+            "legal-api": true,
+            "emailer": false,
+            "filer": false,
+            "entity-bn": true,
+            "digital-credentials": false,
+            "dissolutions-job": false,
+            "furnishings-job": false,
+            "emailer-reminder-job": false,
+            "future-effective-job": false,
+            "update-colin-filings-job": false,
+            "update-legal-filings-job": false
+        }
+    }
+}
```

Comment:
Add a flags.json file to enable entity-bn to get the values of the feature flags